src/libs/Navigation/Navigation.js

```diff
@@ -43,13 +44,33 @@ function navigate(route = ROUTES.HOME) {
 
 /**
  * Dismisses a screen presented modally and returns us back to the previous view.
+ *
+ * @param {Boolean} shouldOpenDrawer
  */
-function dismissModal() {
+function dismissModal(shouldOpenDrawer = false) {
```

Comment:
NAB, just learning, what is the drawer?

Reply:
The main app (anything not a modal) is split into two panels in a [`DrawerNavigator`](https://reactnavigation.org/docs/drawer-navigator/). The left part of the drawer is the "drawer content" and set to the LHN/Sidebar and the right side is the report/chat screen. On larger screens, the drawer is permanently expanded, but on smaller screens / mobile it can be toggled back and forth programmatically or by swiping.

https://github.com/Expensify/Expensify.cash/blob/3dfea0090410aed64199f2e37680429f2682ee0a/src/libs/Navigation/AppNavigator/MainDrawerNavigator.js#L21-L46